tests/moses/scoring/ctruth_table_bscoreUTest.cxxtest

```diff
@@ -36,8 +37,7 @@ using namespace std;
 
 #define GET_VALUE [](const vertex &ver)                              \
 {                                                                    \
-	return ValuePtr(get_builtin(ver)==id::logical_true ?             \
-		   createLink(TRUE_LINK) : createLink(FALSE_LINK));          \
+	return bool_vertex_to_value(ver);                                \
```

Comment:
I think you no longer need `GET_VALUE`, you can directly pass `bool_vertex_to_value` without having to wrap it in a lambda.